src/components/menu-selected-challenge-image.js

```diff
@@ -1,15 +1,12 @@
-import utils from '../utils';
-
 AFRAME.registerComponent('menu-selected-challenge-image', {
   schema: {
-    selectedChallengeId: {type: 'string'}
+    coverURL: { type: 'string' }
```

Comment:
wonder if we can keep passing a `selectedChallengeId` here and handle the URL within the component. Makes the change a bit more surgical and less risk of UI quirks

Reply:
That would make adding a CORS proxy to the URL more complex (needs to be done in two places). Another downside is that the beatsaver API sometimes returns region-specific CDN urls for images (like `eu.cdn.beatsaver.com`) and that would be lost when building the image url from only the selected challenge id